### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zhouyun-zoe**
Add more detailed information.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 # CITA Docs 
 
- Visit the documentation site: [CITA Docs](https://docs.citahub.com/en-US/cita/cita-intro)
+ CITA Docs now has a new independent repo: citahub-docs
```
Comment (ouwenkg) [inline docs/README.md]: add more description to "now" ? like: from the version 0.21 or 2019.3

**Comment by kaikai1024** [comment]
Please skip ci!

**Comment by kaikai1024** [comment]
Why master?

**Comment by kaikai1024** [comment]
Maybe fork it first next time.

**Pull request closed without merging**